### PR TITLE
Author page for @jennaleeb 

### PR DESCRIPTION
**Pull request by mveytsman**

### CODE REVIEW

File: src/authors/jennablumenthal.md [hunk 1]
```diff
@@ -1,14 +1,19 @@
 ---
 name: Jenna Blumenthal
 username: jennablumenthal
-twitter: https://twitter.com/jennaleeblume
+twitter: https://twitter.com/jennaleeb
```
Comment (jennaleeb) [inline src/authors/jennablumenthal.md]: `jennaleeblume` was correct! 😬

File: src/authors/jennablumenthal.md [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
 github: https://github.com/mveytsman
```
Comment (jennaleeb) [inline src/authors/jennablumenthal.md]: This one is `jennaleeb` (i know, different handles, am i even a real programmer?)

File: src/authors/jennablumenthal.md [hunk 3]
```diff
@@ -1,14 +1,19 @@
...
 layout: layouts/author.njk
 ---
 
-# {{name}}
+Jenna Blumenthal has been described as "brilliant", "delightful", and "modest" by her friends and associates. Her modesty has led one of her associates to write this bio for her, making this use of third person entirely natural and warranted.
 
-These are some words about me
+At the Bike Brigade, Jenna manages relationships with some of our partners and works on our [dispatching software](https://github.com/bikebrigade/dispatch), where her focus is on making the experience for our dispatchers and campaign leads as seemless as possible.
```
Comment (jennaleeb) [inline src/authors/jennablumenthal.md]: seemless --> seamless

**Review by jennaleeb — APPROVED** [review]
Your generosity of words make me cringe but also ❤️

File: src/authors/jennablumenthal.md (hunk 1 above)
Comment (mveytsman) [inline src/authors/jennablumenthal.md]:
```suggestion
twitter: https://twitter.com/jennaleeblume
```

File: src/authors/jennablumenthal.md [hunk 4]
```diff
@@ -2,13 +2,18 @@
...
-github: https://github.com/mveytsman
+github: https://github.com/jennaleeb
 layout: layouts/author.njk
 ---
 
-# {{name}}
+Jenna Blumenthal has been described as "brilliant", "delightful", and "modest" by her friends and associates. Her modesty has led one of her associates to write this bio for her, making this use of third person entirely natural and warranted.
 
-These are some words about me
+At the Bike Brigade, Jenna manages relationships with some of our partners and works on our [dispatching software](https://github.com/bikebrigade/dispatch), where her focus is on making the experience for our dispatchers and campaign leads as seemless as possible.
```
Comment (mveytsman) [inline src/authors/jennablumenthal.md]:
```suggestion
At the Bike Brigade, Jenna manages relationships with some of our partners and works on our [dispatching software](https://github.com/bikebrigade/dispatch), where her focus is on making the experience for our dispatchers and campaign leads as seamless as possible.
```

**Pull request merged**